types/expo/index.d.ts

```diff
@@ -2508,21 +2509,30 @@ export namespace SecureStore {
  * Segment
  */
 export namespace Segment {
-    function initialize(keys: {
-        androidWriteKey: string;
-        iosWriteKey: string;
-    }): void;
+    interface SegmentOptions {
+        androidWriteKey?: string;
+        iosWriteKey?: string;
+    }
+
+    function initialize(options: SegmentOptions): void;
+
     function identify(userId: string): void;
     function identifyWithTraits(userId: string, traits: object): void;
-    function track(event: string): void;
-    function alias(newId: string, options?: { [key: string]: any }): Promise<boolean>;
+    function group(groupId: string): void;
+    function groupWithTraits(groupId: string, traits: object): void;
     function reset(): void;
-    function trackWithProperties(event: string, properties: object): void;
+
+    function alias(newId: string, options?: object): Promise<boolean>;
```

Comment:
Why object?
https://github.com/expo/expo/blob/58d73bf85785b74f5659edb1c743e79d72a4b22c/packages/expo-analytics-segment/src/Segment.ts#L49

Reply:
To harmonize the signature :
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/expo/index.d.ts#L2537
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/expo/index.d.ts#L2541
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/expo/index.d.ts#L2543

Reply:
So maybe these should be change too.

https://github.com/expo/expo/blob/58d73bf85785b74f5659edb1c743e79d72a4b22c/packages/expo-analytics-segment/src/Segment.ts#L70

Reply:
Yes :)

Reply:
Want to do this :)?

Reply:
Yes why not :)

Reply:
https://github.com/expo/expo/pull/4116

Reply:
👍